### DISCUSSION TITLE
How to customize window snap to edge behaviour?

### DISCUSSION THREAD

**Discussion by yohandsome** [rxhanson/RectanglePro-Community#40] [Q&A] [answered]
I just bought rectangle pro but can't find a way to customize the window drag snap to behaviour, I'd like that center drag down does center third and that dragging windows to the left/right of the screen fills the window.

Any way to do this?

**Comment by rxhanson** [comment, marked as answer]
Not yet. It's on my roadmap for an upcoming release.

**Comment by yohandsome** [comment]
I'd like to add another use for it, when using a second screen on the left, dragging a window to the left edge doesn't work, but If I could customise snap I could drag it to the lower left edge for instance to cover the left half of the screen.
